src/helpers/sendAppleNotification.ts

```diff
@@ -2,23 +2,37 @@ import { Notification } from '@parse/node-apn'
 import apn from '@/helpers/apn'
 import env from '@/helpers/env'
 
-function createNotification(notificationTitle?: string) {
+function createNotification(
+  notificationTitle?: string,
+  notificationBody?: string
+) {
   const notification = new Notification()
   notification.topic = env.BUNDLE_ID
   notification.contentAvailable = true
   notification.priority = 5
   notification.pushType = 'background'
   if (notificationTitle) {
```

Comment:
Should we also check for `body` here? What if there will be only title, will it be okay without body?